rfcs/text/0000-planning-for-the-dcp.md

```diff
@@ -0,0 +1,209 @@
+
+### Sprint(ing)
+
+During Development, individual developers assign items from the ZenHub *Sprint Backlog* pipeline and move to the *In Progress* pipeline. 
+
+When there is a pull request in review for an *In Progress* item, the item is moved to the *QA/Review* pipeline. 
+
+If rejected, the item returns to *In Progress*; otherwise, it is deployed in Dev for verification and moves to *Done* pipeline.
+
+Items that are *Done* are moved to the *Closed* pipeline when verified in Dev or Integration. 
+
+During Sprints, **Technical Architecture** MAY schedule Scrums on a regular cadence for the DCP Development Team *"to identify if any integration issues exist."* (See *Daily Scrum* in [**Scrum Nexus**](http://scrumorg-website-prod.s3.amazonaws.com/drupal/2018-01/2018-Nexus-Guide-English_0.pdf) for guidance)
+
+**PM** monitors progress using ZenHub charts and reports.
+
+### Sprint Review
+
+**Every four weeks**. On the penultimate day of the Sprint, there is a one hour meeting for the Sprint Review open to the entire DCP commmunity led by *PM*. Major product increments selected by **Technical Leads** are demonstrated to key stakeholders.
+
+(See *Sprint Review* in [*Scrum Guide*](https://scrumguides.org/docs/scrumguide/v2017/2017-Scrum-Guide-US.pdf) and *Nexus Sprint Review* in [*Nexus Guide*](http://scrumorg-website-prod.s3.amazonaws.com/drupal/2018-01/2018-Nexus-Guide-English_0.pdf) for additional details.)
+ 
+### Sprint Retrospective
+
+**Every four weeks**. On the final day of the Sprint, there is a one hour meeting for the Sprint Retrospective open to the entire DCP commmunity led by **PM**.
+ 
+The Scrum Retrospective identifies opportunities to improve the planning process in future iterations. 
+
+(See *Sprint Retrospective* in [*Scrum Guide*](https://scrumguides.org/docs/scrumguide/v2017/2017-Scrum-Guide-US.pdf) and *Nexus Sprint Retrospective* in [*Nexus Guide*](http://scrumorg-website-prod.s3.amazonaws.com/drupal/2018-01/2018-Nexus-Guide-English_0.pdf) for additional details.)
+
+### Acceptance Criteria
+
+This process will be under continuous community introspection and improvement through the Sprint Retrospective. If the process fails to deliver product increments with high confidence after three Sprints, then more radical alternatives should be pursued that satisfy the requirements from our key stakeholders in HCA Science Governance.
+
+### Unresolved Questions
+
+This RFC proposes specific weekdays to start (Thursday) and end (Wednesday) Sprints, but we may need to use a doodle poll to reach consensus. Monday is best avoided due to the number of holidays. 
+
+#### Requested Guidance from Technical Architecture
+- What is the recommended *interpretation* of story points for DCP developers? What does "1" mean? Consider [Engineering Estimates](https://producthabits.com/engineering-estimates/) as an example.
```

Comment:
Suggest story points are relative estimates of effort. Story point velocity would be measured after a few sprints. But I accept actual time estimates are usually inaccurate and misleading, so suggest de-emphasizing their relationship to real time.

Reply:
My perspective is that this should be a Technical Architecture decision. T-shirt sizing is a relative estimate of effort but would still require a standard mapping to the Fibonacci sequence offered by ZenHub story points. I agree that time estimates can be challenging.

Reply:
As an aside you can add any number as a custom zenhub story point, so it doesn't have to be Fibonacci.